commands/create.go

```diff
@@ -513,31 +539,72 @@ func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, machineName, userdataFla
 			return err
 		}
 	}
+	log.Infof("[updateUserdataFile] userdataFile has contents [%s]", userdataFile)
```

Comment:
Can we remove this? User data files often have sensitive information so logging it is not a good idea.

Reply:
Yes, this was present purely for debugging purposes during dev.